app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -578,7 +589,10 @@ fun MapSearchButton(
           ),
       enabled = enabled) {
         Text(
-            text = if (shortText) "Search here" else "Search for hikes",
+            text =
+                if (shortText)
+                    LocalContext.current.getString(R.string.map_screen_search_button_text_short)
+                else LocalContext.current.getString(R.string.map_screen_search_button_text),
```

Comment:
Warning: You should use `stringResources` as in the rest of the code of the app.